commands/cpr

```diff
@@ -111,28 +111,14 @@ action="$(
 
 # Determine available tools for the action
 if test "$action" = 'verify'; then
-	if command-exists rsync; then
-		tools+=(rsync)
-	fi
-	if command-exists diff; then
-		tools+=(diff)
-	fi
-	if command-exists git; then
-		tools+=(git)
-	fi
+	command-exists rsync && tools+=(rsync)
+	command-exists diff  && tools+=(diff)
+	command-exists git   && tools+=(git)
 elif test "$action" = 'copy'; then
-	if command-exists rsync; then
-		tools+=(rsync)
-	fi
-	if command-exists gcp; then
-		tools+=(gcp)
-	fi
-	if command-exists scp; then
-		tools+=(scp)
-	fi
-	if command-exists cp; then
-		tools+=(cp)
-	fi
+	command-exists rsync && tools+=(rsync)
+	command-exists gcp   && tools+=(gcp)
+	command-exists scp   && tools+=(scp)
+	command-exists cp    && tools+=(cp)
```

Comment:
It's common outside Dorothy, but it is not something Dorothy does. It is obtuse (it mixes and matches the roles of conditionals and actions, making groking a more involved process, especially for beginners), and makes modifications such as more conditions and more conditional commands more convoluted.

If bash ever had a linter rewriter (such as prettier.io for other ecosystems), I'd imagine such keystroke shortcut statements would get automatically converted into the expanded if statement upon save as the linter does its rewrites.

Refer to the style guide at https://github.com/bevry/dorothy/discussions/46